sites/public/next.config.js

```diff
@@ -47,8 +40,7 @@ const tailwindVars = require("@bloom-housing/ui-components/tailwind.tosass.js")(
 module.exports = withBundleAnalyzer(
   withTM({
     env: {
-      backendApiBase: BACKEND_API_BASE,
-      listingServiceUrl: BACKEND_API_BASE + LISTINGS_QUERY,
```

Comment:
why get rid of this? seems useful

Reply:
See above re: build-time vs run-time vars.  Keeping both means that you could have different values for `backendApiBase` and `BACKEND_API_BASE` at run time, so I took them out to eliminate any confusing behavior.  From what I can tell, it looks like the build-time vars get plugged into `process.env.prototype`, so even with it set you get some odd behavior around `Object.keys(process.env)` and `process.env.hasOwnProperty('backendApiBase')`.  My personal feelings about libraries opaquely manipulating an object like `process.env` like that aside, I think it makes sense to try to eliminate surprises and duplication.  I actually had a comment on those lines documenting this change in an earlier commit on this branch, but you asked me to remove my comments so I did.  Unless I misunderstood the request, in which case I can add that back.